examples/demos/swarm_demo/control_tower/multi_mav_planning/trajectory_gen/wolfgang_generate_trajectory.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
```

Comment:
This file was used for comparing methods and results while developing. I deleted it since it is not actually used 0d7b8d8e503a92cca3a79d080e0cbe88ff78d1c7